test/unit/Makefile

```
@@ -28,3 +28,16 @@ run:
 		status=$$((status | dir_status)); \
 	done; \
 	exit $$status;
+
+.PHONY	: fclean
+fclean:
+	@status=0; \
+	for dir in $(TEST_DIRS); do \
+		echo "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"; \
+		echo "┃ $$dir"; \
+		echo "┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"; \
+		(MAKEFLAGS="-s" make -C $$dir fclean); \
```

個人的には fclean する時にこの出力部分はいらないかなと
```suggestion
	for dir in $(TEST_DIRS); do \
		(MAKEFLAGS="-s" make -C $$dir fclean); \
```